source/blog/2015-07-15-submission-stats.markdown

```diff
@@ -14,52 +12,55 @@ We had 81 total submissions in 2015!
+If workshops are 3 hours, talks 1 hour, discussions 1 hour, and lightning talks 15 minutes, that's a grand total of **83 hours of content**! We  seriously debated making SassConf two weeks long, in the end we couldn’t pick everything. (Sad trombone.)
 
-**2014 vs 2015**
+## 2014 vs 2015
 
 So how did we stack up compared to last year?
 
 * 78 total -> 81 total
 * 57 unique submitters -> 67 unique submitters
 
-In 2014, 21 of the 78 talks were submitted by someone who had more than one in the running. In 2015, we're down to 14! We love when people submit more than one thing, as it gives us more options, but it's also great to see that we had more total submissions and more unique submitters!
+In 2014, 21 of the 78 talks were submitted by someone who had more than one in the running. In 2015, we're down to 14. We love when people submit more than one thing, as it gives us more options, but it's also great to see that we not only had more total submissions, but also had more **unique** submissions.
+
+
+### Who submitted?
 
+In our earlier post, <a href="/blog/posts/changing-our-cfp/">Why we’re changing our CFP process</a>, we told you that last year we had **less than 5% of submissions from women. **<em>“Women aren’t the only measure of diversity, but it’s the only quantitative data we have from 2014. Another goal for 2015 is to have more data on the SassConf community to share.”</em>
 
-#### Who submitted?
+Our goal was to end up with a 50/50 submission ration and thought we didn’t end up meeting it, we definitely improved over the last two years in this regard.
 
-In our earlier post, <a href="/blog/posts/changing-our-cfp/">Why we’re changing our CFP process</a>, we told you that last year we had **less than 5% of submissions from women.** We also said that <em>“women aren’t the only measure of diversity, but it’s the only quantitative data we have from 2014. Another goal for 2015 is to have more data on the SassConf community to share.”</em>
 
-While we didn’t get a 50/50 _submission_ ratio, we definitely improved on last year in this regard. And we definitely have more data!
+### Here's more on our submission data...*
 
-We had:
+<em>* <small>Disclaimer: These numbers are surface level; based on photos, names, public social media information, and personal knowledge. We cannot guarantee this data is 100% accurate.</small></em>
+
+Of the total 81 submissions in 2015, **67** were from unique submitters. Of those.
 
-* **67** unique submitters. Of those,
 * **48** (72%) were men
-* **19** (28%) were women. 
+* **19** (28%) were women.
 
 > Submissions from women went up to 28%<br/> from last year's 5%!
 
-Thanks, ladies, for making us look so good!
 
-We gleaned some other demographic data about our submitters, as well. 
+We gleaned some other demographic data about our submitters, as well.
 
-* Black Men: **2** 
+* Black Men: **2**
 * Black Women: **1**
 * Asian Men: **4**
 * Asian Women: **1**
-* Latina: **2**
-* LGBTQ+: **5**
+* Latin@: **2**
```

Comment:
@misscs actually, both our submissions were from Latinas, specifically, so the specification was intentional :)